### PR TITLE
8331582: Incorrect default Console provider comment

### PR DESCRIPTION
**Pull request by naotoj**
Fixing a comment in `java.io.Console`, which was a leftover from the fix to https://bugs.openjdk.org/browse/JDK-8308591

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8331582](https://bugs.openjdk.org/browse/JDK-8331582): Incorrect default Console provider comment (**Bug** - P4)

### Reviewers
 * [Joe Wang](https://openjdk.org/census#joehw) (@JoeWang-Java - **Reviewer**) ⚠️ Review applies to [2ab4fdf9](https://git.openjdk.org/jdk/pull/19070/files/2ab4fdf952b6ae4224b8c0d95d7de68fb964fa3c)
 * [Jan Lahoda](https://openjdk.org/census#jlahoda) (@lahodaj - **Reviewer**) ⚠️ Review applies to [2ab4fdf9](https://git.openjdk.org/jdk/pull/19070/files/2ab4fdf952b6ae4224b8c0d95d7de68fb964fa3c)
 * [Justin Lu](https://openjdk.org/census#jlu) (@justin-curtis-lu - Committer) ⚠️ Review applies to [2ab4fdf9](https://git.openjdk.org/jdk/pull/19070/files/2ab4fdf952b6ae4224b8c0d95d7de68fb964fa3c)
 * [Pavel Rappo](https://openjdk.org/census#prappo) (@pavelrappo - **Reviewer**) ⚠️ Review applies to [2ab4fdf9](https://git.openjdk.org/jdk/pull/19070/files/2ab4fdf952b6ae4224b8c0d95d7de68fb964fa3c)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/19070/head:pull/19070` \
`$ git checkout pull/19070`

Update a local copy of the PR: \
`$ git checkout pull/19070` \
`$ git pull https://git.openjdk.org/jdk.git pull/19070/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 19070`

View PR using the GUI difftool: \
`$ git pr show -t 19070`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/19070.diff">https://git.openjdk.org/jdk/pull/19070.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/19070#issuecomment-2091568035)

### CODE REVIEW

**Review by JoeWang-Java — APPROVED** [review]

**Review by lahodaj — APPROVED** [review]
Looks good to me, thanks!

**Review by justin-curtis-lu — APPROVED** [review]

File: src/java.base/share/classes/java/io/Console.java [hunk 1]
```diff
@@ -406,7 +406,8 @@ private static Console instantiateConsole(boolean istty) {
             /*
              * The JdkConsole provider used for Console instantiation can be specified
              * with the system property "jdk.console", whose value designates the module
-             * name of the implementation, and which defaults to "java.base". If no
+             * name of the implementation, and which defaults to the value
```
Comment (pavelrappo) [inline src/java.base/share/classes/java/io/Console.java:409]: Might read slightly better:
```suggestion
             * name of the implementation, and which defaults to the value of
```

**Review by pavelrappo — APPROVED** [review]
Looks good.

**Comment by naotoj** [comment]
/integrate
Thank you for the reviews!

**Pull request closed without merging**